spec/requests/api/v1/catches_request_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Flies request API' do
+  before:each do
+    @fly1 = create(:fly)
+    @catch1 = create(:catch)
+    @catch2 = create(:catch)
+    @catch3 = create(:catch)
+    @catch4 = create(:catch)
+  end
+
+  it "can return all catches" do
+    get '/api/v1/catches'
+    expect(response).to be_successful
+    catches = JSON.parse(response.body)
+    expect(catches["data"].length).to eq(4)
+  end
+
+  it "can delete a particular catch by id" do
+    delete "/api/v1/catches/#{@catch4.id}"
+    expect(response).to be_successful
+    expect(Catch.all.count).to eq(3)
+    expect{Catch.find(@catch4.id)}.to raise_error(ActiveRecord::RecordNotFound)
+  end
+
+  it "can create a new catch with attributes and a fly id" do
+    new_catch_params = {species: "New Species", location: "Moms House", length: 111, weight: 45, fly_id: @fly1.id}
+    post "/api/v1/catches", params: {catch: new_catch_params}
```

Comment:
If `.require` is remove on controller can remove the hash and just make `new_cath_params` in parentheses